test-cases/longevity/longevity-2TB-48h-authorization-and-tls-ssl-1dis-2nondis-nemesis.yaml

```diff
@@ -29,6 +29,7 @@ instance_type_loader: 'c5.2xlarge'
 
 cluster_health_check: false
 nemesis_class_name: 'DisruptiveMonkey:1 NonDisruptiveMonkey:2'
+nemesis_selector: [[],[],["!manager_operation"]]
```

Comment:
Did you test it?
I'm surprised it works in this format.
Anyway, if you test it, It's approved.

Reply:
I know it works in this format

And we have several tests using it like that

And even have a unit test for those

Reply:
@fruch Here you define 3 nemesis selectors for 2 nemesis threads. Why?
Why not 2 nemesis selectors?

Reply:
It expands to 3 here:

https://github.com/scylladb/scylla-cluster-tests/blob/53cd2d2662565307007e2de2ce4765a9bf30c831/sdcm/cluster.py#L4276

But yeah this won't work as I expected, small refactoring in needed, so each one would be able to have it's own selector